semconv/v1_4_0/doc.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package semconv implements OpenTelemetry semantic conventions start from specification
+// version 1.4.0 and until the next version that introduced any changes.
```

Comment:
```suggestion
// Package semconv/v1_4_0 implements v1.4.0 of the OpenTelemetry semantic conventions.
```

Reply:
is that valid? Or should it be "Package v1_4_0 implements..."?

Reply:
Not entirely sure. I was going off this https://pkg.go.dev/golang.org/x/sys/unix

> The sys/unix package provides...

I'm go with the other way as well.